application/binary/build.gradle.kts

```diff
@@ -1,6 +1,8 @@
 androidBinary(
     packageName = "com.stepango.blockme.app",
     owner = Teams.core,
+    versionCode = 1,
```

Comment:
I think, better to move out BuildConfig definitions from android binaries and just passing to them already created, immutable version of type configuration: DebugBuildConfig, ReleaseBuildConfig and etc.